src/components/Quote.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import '../style/Quote.css';
+
+export default function Quote() {
+  const [quote, setQuote] = useState('loading quote');
+  const [loading, setLoading] = useState('loading');
+  const fetchQuote = async () => {
+    try {
+      const response = await fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+        headers: { 'X-Api-Key': 'nezpEokeOVTwP1C5NwtGEQ==h62LjJXfJNexoQfy' },
+      });
```

Comment:
- [ ]  Good job for using the provided API correctly and successfully. One thing I recommend is not only including the headers but also specifying the method you've used, which is `GET` in your case. This will improve the readability of your code, improve proper functionality of the api in your api, clear comminication, etc.